### PR TITLE
f5fpc: add page

### PR DESCRIPTION
**Pull request by sbrl**
My university forces us to use this VPN client if we want VPN access, so I thought I'd document it here so I don't have to look through the documentation again :P

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by sbrl** [comment]
Huh, ok. Didn't realise it was required! It asks you if you don't specify them. I'll add them now.

**Comment by sbrl** [comment]
Erm I have literally no clue what you're complaining about, @tldr-bot.

**Comment by pxgamer** [comment]
Yeh, I can't see anything wrong. :thinking: Unless it's a line-ending issue? But surely not...

File: pages/linux/f5fpc.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+- Open a new VPN connection:
+
+`f5fpc --start`
+
+- Open a new VPN connection to a specific host:
+
+`f5fpc --start --host {{host.example.com}}` 
+
+- Specify a username:
```
Comment (agnivade) [inline pages/linux/f5fpc.md]: I guess it prompts you for the password ? Should we mention it here ?

Comment (sbrl) [reply pages/linux/f5fpc.md]: Ah, good idea.

**Comment by sbrl** [comment]
Thanks so much, @pxgamer! It would have taken me ages to spot that :P

**Pull request merged**